Release notes 1.8.0 / 2.11.0 (#6294)
voxel51/fiftyone: docs/source/plugins/using_plugins.rst

box with a connected compute cluster for executing delegated operations at
scale.

.. note::

>     [ Beta ] FiftyOne Enterprise now supports on-demand compute via Databricks
>     and Anyscale. In order to configure an orchestrator for your Databricks or
>     Anyscale compute, consult your FiftyOne Enterprise Helm Chart docs.

Reviewer: I would either link directly to deployment docs, or link to <enterprise-delegated-orchestrator> which is the "Configuring your orchestrator" section for Enterprise.
```suggestion
    [ Beta ] FiftyOne Enterprise now supports on-demand compute via Databricks
    and Anyscale. In order to configure an orchestrator for your Databricks or
    Anyscale compute, administrators should consult the
    `deployment guide <https://github.com/voxel51/fiftyone-teams-app-deploy/blob/main/docs/configuring-on-demand-orchestrator.md>`_
```

Author: Do we ever include links to the deployment docs? I couldn't find any examples of that. 

Also, why link to the github repo instead of https://helm.fiftyone.ai/ ?

Reviewer: Hmm I guess idk? Is there an equivalent to that for docker compose customers?
